tests/src/test/scala/whisk/core/controller/test/EntitlementProviderTests.scala

```diff
@@ -534,25 +536,27 @@ class EntitlementProviderTests extends ControllerTestCommon with ScalaFutures {
     implicit val tid = transid()
     implicit val ep = entitlementProvider
 
+    val provider = WhiskPackage(someUser.namespace.toPath, MakeName.next(), None, publish = true)
+    val binding = WhiskPackage(guestUser.namespace.toPath, MakeName.next(), provider.bind)
+    val action = WhiskAction(binding.fullPath, MakeName.next(), jsDefault(""))
+    put(entityStore, provider)
+    put(entityStore, binding)
+    put(entityStore, action)
+
+    val resourceName = s"${binding.namespace}/${binding.name}"
```

Comment:
I suggest using `binding.fullyQualifiedName(false).asString` instead here and elsewhere a name is constructed explicitly.